Libraries/CommonFunctions.psm1

```diff
@@ -1739,7 +1739,7 @@ Function Start-StressNg {
         -username $user -password $password -upload
     # execute command as job
     $retVal = Run-LinuxCmd -username $user -password $password -ip $VMIpv4 -port $VMSSHPort `
-        -command "echo $password | cd /home/$user && chmod u+x ${FILE_NAME} && sed -i 's/\r//g' ${FILE_NAME} && ./${FILE_NAME}" -runAsSudo
+        -command "echo $password | cd /home/$user && chmod u+x ${FILE_NAME} && sed -i 's/\r//g' ${FILE_NAME} && ./${FILE_NAME}" -RunInBackground
```

Comment:
I don't understand the reason for change this.

Reply:
The test logic is the demand memory is increasing during the memory stress test. Before the change, the Start-StressNg is not only to start the stress test , but also wait it finish. What we excepted is that reading the demand memory during the memory stress is ongoing. Otherwise, the demand memory may equal to $vm1AfterStressDemand in a high rate.

Reply:
It should be also runAsSudo.

Reply:
From my test, the runAsSudo is not needed, but keeping it maybe better.